api-contracts/v1/shared/condition.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+option go_package = "github.com/hatchet-dev/hatchet/internal/services/shared/proto/v1";
+
+package v1;
+
+enum Action {
+    CREATE = 0;
+    QUEUE = 1;
```

Comment:
Document: QUEUE, CANCEL, SKIP relevant for task triggers, CREATE relevant for signals